platform.md

```diff
@@ -1089,16 +1236,40 @@ For more information on build reproducibility see [https://reproducible-builds.o
 # layer metadata
 
 [run-image]
+  image = "<image>"
   reference = "<image reference>"
+  extend = false
+  [target]
+  id = "<target identifer>"
+  os = "<OS name>"
+  arch = "<architecture>"
+  variant = "<architecture variant>"
+  [target.distribution]
```

Comment:
Is it weird that the labels are `distro`, but the it's `distribution` here?

Reply:
Whoops... I think I just missed this.